metadata/political/vista_ballot_areas.md

```diff
@@ -0,0 +1,75 @@
+# Title
+
+Utah VISTA Ballot Areas
+
+## ID
+
+fb485021-1a93-43b3-bd3b-13c68d7e5b3c
+
+## Brief Summary
+
+Official voting precincts used by Utah's VISTA election framework.
+
+## Summary
+
+Statewide polygon layer that depicts voting precinct and subprecinct boundaries. This dataset serves as the baseline for the Voter Information and State Tracking Application (VISTA), as well as many other voter information websites.
+
+## Description
+
+### What is the dataset?
+
+Voting precincts are used to determine a voter's ballot composition and polling location. A number of factors contribute to this, including the boundaries of all political districts covering the voter's address (U.S. Congress, State House and Senate, State School Board, local county and municipal districts, etc.) and the legislatively-mandated maximum limit of voters per precinct.
+
+### What is the purpose of the dataset?
+
+This dataset is actively used in all 29 counties in Utah. The counties have adopted a GIS-based approach to manage precinct-to-residence assignment within VISTA, the statewide voter registration database. This dataset also drives many of the State's voter information pages and helps citizens know [how and where to vote](https://votesearch.utah.gov/voter-search/search/search-by-address/how-and-where-can-i-vote).
+
+### What does the dataset represent?
+
+This dataset contains polygons representing more than 3000 voting precincts and subprecincts in Utah. Each feature contains attributes such as the date the precinct boundary took effect (effectivedate) and whether the polygon contains a subprecinct boundary (indicated, where applicable, in the subprecinct field).
+
+<!--- This next paragraph is from the original metadata. I haven't seen other data pages explain GIS operations for a dataset in this way, is this something we generally like to include? -->
+
+Precincts can be formed (and subprecinct divisions eliminated) by performing a GIS ‘dissolve’ operation on the precinctid field.
```

Comment:
```suggestion
Precincts can be formed by eliminating subprecinct divisions by dissolving the features on the precinctid field.
```
I think this is important, as it talks about how you can derive precincts from the dataset. Maybe we just make it a little less "tutorial-ly"?